Lib/multiprocessing/managers.py

```diff
@@ -8,7 +8,8 @@
 # Licensed to PSF under a Contributor Agreement.
 #
 
-__all__ = [ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token' ]
+__all__ = [ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token',
+            'SharedMemoryManager' ]
```

Comment:
According to [this comment]( https://bugs.python.org/issue35813#msg338180 ), sounds like adding `SharedMemoryManager` to `__all__` should be handled to the `try...except` below.

PS: Nice work on this btw!